org.nongnu.gsequencer.gsequencer.json

```diff
@@ -0,0 +1,237 @@
+                },
+                {
+                    "type": "script",
+                    "commands": [
+                        "autoreconf -fiv"
+                    ],
+                    "dest-filename": "autogen.sh"
+                }
+            ]
+        },
+
+        {
+            "name": "audiofile",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://audiofile.68k.org/audiofile-0.3.6.tar.gz",
+                    "sha256": "cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/audiofile-gcc6.patch"
+                }
+            ]
+        },
+        
+        {
+            "name": "libinstpatch",
+            "rm-configure": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/swami/libinstpatch/1.0.0/libinstpatch-1.0.0.tar.gz",
+                    "sha256": "8943e2226d97609104365f0105be87b326a0b2335cea22d29604a45f8ef1cee7"
+                },
+                {
+                    "type": "patch",
+                    "strip-components": 0,
+                    "path": "patches/instpatch.examples.patch"
+                },
+                {
+                    "type": "script",
+                    "commands": [
+                        "autoreconf -fiv"
+                    ],
+                    "dest-filename": "autogen.sh"
+                }
+            ]
+        },
+        
+        {
+            "name": "lv2",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./waf configure --prefix=/app --docdir=/app/share/doc/lv2",
+                "./waf build -j $FLATPAK_BUILDER_N_JOBS",
+                "./waf install"
+            ],
+            "cleanup": [
+                "/bin",
+                "/share",
+                "/lib/jack",
+                "/lib/libjackserver.so*"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://lv2plug.in/spec/lv2-1.16.0.tar.bz2",
+                    "sha256": "dec3727d7bd34a413a344a820678848e7f657b5c6019a0571c61df76d7bdf1de"
+                }
+            ]
+        },
+
+	{
+            "name": "chrpath",
+	    "cleanup": ["*"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://src.fedoraproject.org/lookaside/extras/chrpath/chrpath-0.16.tar.gz/2bf8d1d1ee345fc8a7915576f5649982/chrpath-0.16.tar.gz",
+                    "sha256": "bb0d4c54bac2990e1bdf8132f2c9477ae752859d523e141e72b3b11a12c26e7b"
+                }
+            ]
+	},
+	
+        {
+            "name": "gsequencer",
+            "config-opts": [
+                "BUILDCC=/usr/bin/gcc -std=gnu99"
```

Comment:
This is odd. `gcc` is surely your default and if your application wants `-std=gnu99` it should be setting that itself.

Reply:
It does it.

Reply:
So remove it?